docs-new/docs/graphql/core/scheduled-triggers/create-one-off-scheduled-event.mdx

```diff
@@ -0,0 +1,158 @@
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+In the form opened by the above step, fill out the following fields:
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/define-one-off-event.png"
+  alt="Defining the scheduled event"
+  width="550px"
+/>
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+One-off scheduled events cannot be set using the CLI as they are not tracked as a part of Hasura metadata.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+You can define a scheduled event via the [create_scheduled_event](/graphql/core/api-reference/metadata-api/scheduled-triggers.mdx#metadata-create-scheduled-event) metadata API:
+
+```http
+POST /v1/metadata HTTP/1.1
+Content-Type: application/json
+X-Hasura-Role: admin
+
+{
+   "type": "create_scheduled_event",
+   "args": {
+      "webhook": "https://send-email.com",
+      "schedule_at": "2022-06-18T18:45:00Z",
+      "payload": { "email": "bob@ross.com" }
+   }
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Step 3: Define advanced options (Optional)
+
+If you like, you can also define advanced values:
+
+-   **Headers**: List of headers to be sent to the webhook.
+-   **Retry configuration**: In case the call to the webhook fails.
+-   **Comment**: Custom description of the scheduled trigger.
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+
+Expand the `Advanced` section.
+
+<Thumbnail
+  src="/img/graphql/core/event-triggers/advanced-one-off.png"
+  alt="Defining the scheduled event"
+  width="700px"
+/>
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+One-off scheduled events cannot be set using the CLI as they are not tracked as a part of Hasura metadata.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+You can define advanced options when defining a scheduled event via the [create_scheduled_event](/graphql/core/api-reference/metadata-api/scheduled-triggers.mdx#metadata-create-scheduled-event) metadata API:
+
+```http
+POST /v1/metadata HTTP/1.1
+Content-Type: application/json
+X-Hasura-Role: admin
+
+{
+   "type": "create_scheduled_event",
+   "args": {
+      "webhook": "https://send-email.com",
+      "schedule_at": "2022-06-18T18:45:00Z",
+      "payload": {
+            "email": "bob@ross.com"
+      },
+      "headers": [
+            {
+               "name": "key",
+               "value": "value"
+            }
+      ],
+      "retry_conf": {
+            "num_retries": 3,
+            "timeout_seconds": 120,
+            "tolerance_seconds": 21675,
+            "retry_interval_seconds": 12
+      },
+      "comment": "sample scheduled event comment"
+   }
+}
+```
+
+</TabItem>
+</Tabs>
+
+## Schedule & logs
+
+Once you've created your scheduled trigger, you can see `Pending events`, `Processed events`, and `Invocation logs` in their respective tabs.
+
+<Thumbnail
+  src="/img/graphql/cloud/read-replicas/connect-db-with-replica.png"
+  alt="Connect database with read replica"
+  width="1200px"
+/>
+
```

Comment:
Mismatching image - please refer original file and update correct onw

Reply:
Done